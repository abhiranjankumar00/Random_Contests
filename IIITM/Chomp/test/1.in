100
9 6 2
6 1 1
32 6 1
32 8 4
21 3 3
22 15 2
3 0 0
29 27 9
3 2 2
14 7 7
20 0 0
34 24 17
9 6 2
3 1 1
27 7 0
18 9 4
31 18 16
27 7 4
2 0 0
26 22 3
31 29 14
13 10 9
27 25 11
8 0 0
3 1 0
3 2 2
24 0 0
4 1 0
19 4 3
17 8 1
13 4 1
29 19 12
12 12 6
15 11 10
25 2 2
27 21 18
18 0 0
2 1 0
7 1 1
35 28 12
6 3 3
33 11 11
32 2 0
13 11 4
13 5 0
1 0 0
5 2 1
32 17 17
4 4 0
4 3 3
11 9 6
19 18 8
7 3 1
1 1 0
10 8 1
11 1 0
20 16 14
25 7 0
29 21 13
16 1 0
3 3 1
14 5 5
20 1 0
31 29 7
16 4 3
34 33 31
19 9 6
35 7 3
14 8 5
31 24 18
29 18 18
13 11 0
15 15 14
9 0 0
18 18 0
21 13 9
30 14 9
13 4 4
13 1 1
35 20 2
23 12 7
34 1 0
6 6 1
26 24 1
2 0 0
1 0 0
10 9 2
26 23 16
3 2 2
3 2 2
31 4 2
30 21 2
4 1 1
17 9 0
19 14 10
24 24 6
20 19 19
14 13 2
6 2 2
32 26 24
