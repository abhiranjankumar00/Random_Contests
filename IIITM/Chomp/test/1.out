WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
LOSE
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
LOSE
WIN
WIN
LOSE
WIN
WIN
WIN
WIN
LOSE
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
LOSE
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
WIN
